commit d14d37193873888b207a992abe4a5848a5023ff6
Author: popcornmix <popcornmix@gmail.com>
Date:   Mon Oct 15 18:54:45 2012 +0100

    Revert "Remove BUG() call from vchiq that is trigger when alsa is killed"
    
    This reverts commit 8bccc7a0340404cc14f831dd9c06d3528b5ebcff.

Index: linux-3.2.46/drivers/misc/vc04_services/interface/vchiq_arm/vchiq_core.c
===================================================================
--- linux-3.2.46.orig/drivers/misc/vc04_services/interface/vchiq_arm/vchiq_core.c	2013-07-26 19:37:01.000000000 +0000
+++ linux-3.2.46/drivers/misc/vc04_services/interface/vchiq_arm/vchiq_core.c	2013-07-26 19:37:05.000000000 +0000
@@ -3180,7 +3180,7 @@
 		mode, VCHIQ_BULK_TRANSMIT);
 
 	/* This call is for kernel thread use and should not be interrupted */
-	// dc4: remove as it does happen: BUG_ON(status == VCHIQ_RETRY);
+	BUG_ON(status == VCHIQ_RETRY);
 	return status;
 }
 
